coalib/parsing/DefaultArgParser.py

```diff
@@ -115,6 +115,16 @@ def default_arg_parser(formatter_class=None):
         '--flush-cache', const=True, action='store_const',
         help='rebuild the file cache')
 
+    if parser_type == "coala-format":
+        config_group.add_argument(
+            '--format-str', const=True, action='store', nargs='+',
+            metavar='FIELD',
+            help='To format the printed text by giving one or more of the '
```

Comment:
shorten the message and start lower case, read through the others. Help texts have to be very consistent and compressed to the most necessary info only so people can read them quickly